<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QMainWindow" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="gameMode">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Lokales Spiel</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Online Spiel</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createLocalGame"/>
    <widget class="QWidget" name="remoteGameChoice">
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Beitreten</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Erstellen</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="joinRemoteGame">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000.000.000.000;_</string>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="placeholderText">
       <string>Server IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>DDDDDD</string>
      </property>
      <property name="text">
       <string>1337</string>
      </property>
      <property name="placeholderText">
       <string>Server Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Beitreten</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zur√ºck</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
